### ISSUE TITLE
Cannot enable ingress add-on with K8s v1.18.20 and Minikube v1.23.2

### ISSUE THREAD

**Issue by AlexIoannides** [kubernetes/minikube#12636]
With Minikube v1.23.2 running on macOS v11.6 installed via homebrew (commit: 0a0ad764652082477c00d51d2475284b5d39ceed), I get the following error when I try to enable the ingress add-on with an old version of Kubernetes:

1. minikube start --kubernetes-version=v1.18.20
2. minikube addons enable ingress

```
error: unable to recognize "/etc/kubernetes/addons/ingress-deploy.yaml": no matches for kind "IngressClass" in version "networking.k8s.io/v1"
```

I don't think that `networking.k8s.io/v1` is part of the v1.18.2 k8s API, which may be the source of the issue?

**Comment by sharifelgamal** [comment]
Yep, that's definitely the issue. We switched to networking.k8s.io/v1 to support k8s 1.22 and beyond, and that has inadvertently broken ingress for older versions of k8s. 

We do want to support older k8s versions for ingress (I think we might even have some code that attempts it), but for now the best workaround is to either downgrade your version of minikube or upgrade your version of k8s.

**Comment by AlexIoannides** [comment]
Thanks Sharif - appreciated. 

The problem we have is that the [Kubernetes Python client](https://github.com/kubernetes-client/python) is currently at v1.18. I'll look into downgrading Minikube.
